tests/core/test_lifetime_runnable.py

```diff
@@ -1,24 +1,19 @@
-from typing import Type
 from unittest.mock import MagicMock
 
 import pytest
 
-from tickit.core.lifetime_runnable import LifetimeRunnable, run_all_forever
+from tickit.core.lifetime_runnable import run_all_forever
 
 
-@pytest.fixture
-def TestLifetimeRunnable():
-    class TestLifetimeRunnable:
-        async def run_forever(self) -> None:
-            while True:
-                pass
-
-    return TestLifetimeRunnable
+class TestLifetimeRunnable:
+    async def run_forever(self) -> None:
+        while True:
+            pass
```

Comment:
What's the thought behind removing this class definition from a fixture? Being inside one protects it from mutations in other tests (obviously not a problem here, but afaik considered good practice)